tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/OrdersControllerTests.cs

```diff
@@ -566,9 +568,10 @@ private static (Order order, OrderModel expectedOrder) CreateGetTestData(string
                     Address = repositoryOrder.SupplierAddress.ToModel(),
                     PrimaryContact = repositoryOrder.SupplierContact.ToModel()
                 },
-                TotalOneOffCost = repositoryOrder.CalculateCostPerYear(CostType.OneOff),
+                TotalOneOffCost = totalOneOffCost,
                 TotalRecurringCostPerMonth = calculatedCostPerYear / monthsPerYear,
                 TotalRecurringCostPerYear = calculatedCostPerYear,
+                TotalOwnershipCost = totalOneOffCost + (defaultContractLength * calculatedCostPerYear),
```

Comment:
Not sure about duplicating the logic in the test. Would it be possible to assert the expected value instead?

Reply:
Using the calculation from the Order.cs